src/status_im/ui/screens/activity_center/notification/contact_verification/view.cljs

```diff
@@ -0,0 +1,105 @@
+(ns status-im.ui.screens.activity-center.notification.contact-verification.view
+  (:require [clojure.string :as str]
+            [quo2.core :as quo2]
+            [status-im.constants :as constants]
+            [status-im.i18n.i18n :as i18n]
+            [status-im.multiaccounts.core :as multiaccounts]
+            [status-im.ui.screens.activity-center.notification.contact-verification.style :as style]
+            [status-im.ui.screens.activity-center.utils :as activity-center.utils]
+            [status-im.utils.datetime :as datetime]
+            [status-im.utils.re-frame :as rf]))
+
+(defn- hide-bottom-sheet-and-dispatch
+  [event]
+  (rf/dispatch [:bottom-sheet/hide])
+  (rf/dispatch [:dismiss-keyboard])
+  (rf/dispatch event))
```

Comment:
Can `dispatch-n` be used here ?

Reply:
Good suggestion, but I believe we can't use it, because this is only available as an effect, and not as a function. `re-frame.core/dispatch-n` doesn't exist as far as I checked.